vtx_common/tools/replace_commit.py

```diff
@@ -0,0 +1,34 @@
+import sys
+import subprocess
+
+# Replace the embedded commit information with our current git commit
+# Takes the file path as a single argument.
+
+def main(argv):
+    filepath = argv[0]
+    try:
+        ret = subprocess.run(['git', 'rev-parse', 'HEAD'],
+                             capture_output=True,
+                             timeout=15,
+                             check=False,
```

Comment:
Since you have `check=False`, I don't think this will ever raise.

I'd recommend using `check_output()` instead of `run()`

Reply:
It is still possible to raise ( example: if `git` is not present ); it just won't raise based on return code.

Reply:
changed to check=True; subprocess.run is the currently reccomended high level api for that module